src/utils/dom/querySelectorAll.js

```diff
@@ -12,5 +12,22 @@ governing permissions and limitations under the License.
 
 import toArray from "../toArray";
 
-export default (context, selector) =>
-  toArray(context.querySelectorAll(selector));
+const SIBLING_PATTERN = /^\s*>/;
+
+export default (context, selector) => {
+  if (!SIBLING_PATTERN.test(selector)) {
+    return toArray(context.querySelectorAll(selector));
+  }
+
+  const tag = `alloy${+new Date()}`;
+
+  try { // adding a dummy css class to be able to select the children
```

Comment:
// We could use a :scope selector here, but we want to be IE compliant
// so we add a dummy css class to be able to select the children